apps/rpicam_vid.cpp

```diff
@@ -85,6 +85,18 @@ static void event_loop(RPiCamEncoder &app)
 	signal(SIGPIPE, default_signal_handler);
 	pollfd p[1] = { { STDIN_FILENO, POLLIN, 0 } };
 
+	libcamera::ControlList cl;
+	if (options->sync == 0)
+		cl.set(libcamera::controls::rpi::SyncMode, libcamera::controls::rpi::SyncModeOff);
+	else if (options->sync == 1)
+		cl.set(libcamera::controls::rpi::SyncMode, libcamera::controls::rpi::SyncModeServer);
+	else if (options->sync == 2)
+		cl.set(libcamera::controls::rpi::SyncMode, libcamera::controls::rpi::SyncModeClient);
+	app.SetControls(cl);
```

Comment:
Should we add this to the the core code?  Then it can be used by ``rpicam-raw``  as well if needed?